recipe_batches/recipe_batches.py

```diff
@@ -2,13 +2,29 @@
 
 import math
 
+
 def recipe_batches(recipe, ingredients):
-  pass 
+    count = 0
+    while True:
+        for item in recipe:
+            if item not in ingredients:
+                return count
+            elif ingredients[item] < recipe[item]:
+                return count
+            else:
+                ingredients[item] -= recipe[item]
+        # If each item in recipe gets to the else statement
```

Comment:
Nice this is a pretty efficient solution. Keep up the comments it will help you along in your career.